src/ru/help/guides/getting-started.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Как начать
+lang: ru-RU
+---
+
+# Как начать
+
+## Установка
+**Paperback** в данный момент _не_ на **TestFlight** и _требует_ использование **AltStore** что бы установить на ваш телефон. В будущем, TestFlight будет использован когда главный программист получит [Apple Developer Account](https://developer.apple.com/programs/).
+
+::: tip Видео руководство
+ Если вы предпочитаете видео руководство, Artesians создал очень хорошее видео (на английском). Вы сможете найти его [здесь для Windows](https://www.youtube.com/watch?v=n1KRwsxNiWY) и [здесь для Mac](https://www.youtube.com/watch?v=CjPjsF4yJ0M).
+:::
+
+### Необходимые условия
+1. На вашем телефоне, скачайте приложение через главную страницу этого вэб-сайта. Приложение будет скачано как **`Paperback.ipa`**.
+	
+	Альтернативно, вы можете скачать файл через нашу [страницу GitHub releases](https://github.com/FaizanDurrani/Paperback/releases/latest).
```

Comment:
The download button was removed from the main page. You should use:
```suggestion
 1. On your phone, download the app <Download text="from here"/>. The app will download as **`Paperback.ipa`**.
```

Reply:
Since it’s removed there’s nothing to link to anymore.

Reply:
`<Download text="from here"/>` will create a link to the last release.